### ISSUE TITLE
Strange typesetting of words "GPL" and "LGPL" in license text

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#4634]
## Issue Details

When looking at the file https://doc.cgal.org/latest/Manual/license.html we see here that the words "GPL" and "LGPL" are set in a, in my opinion, a bit a strange way.
![CGAL_lic](https://user-images.githubusercontent.com/5223533/78889469-fa6cb280-7a63-11ea-947e-bf3b1f4536e8.jpg)

## Source Code

The origin of this problem is the fact that the wordds "GPL" and "LGPL" are set in so called "small-caps" and that in the code they are written as: `\sc{Gpl}` and `\{Lgpl}`:
```
Documentation/doc/Documentation/License.txt:6:\sc{Gpl}/\sc{Lgpl} open source license, as well as under commercial licenses.
Documentation/doc/Documentation/License.txt:19:The \sc{Gpl} is an Open Source license that, if you distribute your software
Documentation/doc/Documentation/License.txt:20:based on \sc{Gpl}ed \cgal data structures, obliges you to distribute the
Documentation/doc/Documentation/License.txt:21:source code of your software under the \sc{Gpl}.
Documentation/doc/Documentation/License.txt:28:The \sc{Lgpl} is an Open Source license that obliges you to distribute
Documentation/doc/Documentation/License.txt:30:In contrast to the \sc{Gpl}, there is no obligation to make the source
Documentation/doc/Documentation/License.txt:31:code of software you build on top of \sc{Lgpl}ed \cgal data structures.
Documentation/doc/Documentation/License.txt:38:We have chosen the \sc{Gpl} and the \sc{Lgpl} as they are well-known
Documentation/doc/Documentation/License.txt:44:the \sc{Lgpl}, and the higher level packages under the \sc{Gpl}.
Documentation/doc/Documentation/License.txt:60:use packages of \cgal released under the \sc{Lgpl}.
```

I think that it probably is better to use `\sc{GPL}` and `\sc{LGPL}`.

I haven't, yet, made a pull request of it as I think it is a a bit an opinion problem.

**Comment by albert-github** [comment]
This issue has been (automatically) closed but I still found:
- Documentation/doc/Documentation/License.txt line 6

**Comment by sloriot** [comment]
Fixed in d03a02368dc. Thanks!
